ws/config.py

```diff
@@ -15,33 +15,31 @@
     "getArgParser", 
     "object_from_argparser",
     "argtype_bool",
-    "argtype_configfile",
+    "argtype_config",
     "argtype_path",
 ]
 
-PROJECT_NAME = 'wiki-scripts'
-CONFIG_DIR = '~/.config/'       # Can be overriden by XDG_CONFIG_HOME
-CACHE_DIR = '~/.cache/'         # Can be overriden by XDG_CACHE_HOME
-DEFAULT_CONF = 'default'        # Can be overriden with '--config' option
+PROJECT_NAME = "wiki-scripts"
+CONFIG_DIR = os.path.expanduser("~/.config/")
+CACHE_DIR = os.path.expanduser("~/.cache/")
+DEFAULT_CONF = "default"
 
 
-# ==============
-# Parser classes
-# ==============
-
 class ConfigParser(configparser.ConfigParser):
     """Drop-in replacement for :py:class:`configparser.Configparser`."""
 
-    def __init__(self, configfile):
-        super().__init__()
+    def __init__(self, configfile, **kwargs):
+        kwargs.setdefault('interpolation',
+                          configparser.ExtendedInterpolation())
```

Comment:
Please don't break this on multiple lines - it looks ugly.

Reply:
As you might have noticed, I don't really like PEP8 - I find some of its style conventions really stupid, especially the 79 chars per line limit. It's 2020! All major code editors can do soft wrapping with nice indentation. Even without visual indentation soft-wrapped code looks much better than soft-wrapping applied on hard-wrapped code.

Reply:
But PEP 8... 79 characters on a line...